drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -287,9 +287,37 @@ static void esp_wifi_handle_sta_disconnect_event(void *event_data)
 static void esp_wifi_handle_ap_connect_event(void *event_data)
 {
 	wifi_event_ap_staconnected_t *event = (wifi_event_ap_staconnected_t *) event_data;
+	wifi_sta_list_t sta_list;
+	struct wifi_ap_sta_info sta_info;
+
+	sta_info.link_mode = WIFI_LINK_MODE_UNKNOWN;
+	memcpy(sta_info.mac, event->mac, WIFI_MAC_ADDR_LEN);
+	sta_info.mac_length = WIFI_MAC_ADDR_LEN;
+	sta_info.twt_capable = false;           /* Only support in 802.11ax */
```

Comment:
another way would be to just `memset` the `sta_info` and only fill in non-default value.

Reply:
Considering that most of the data is non-zero except for `twt_capable`, `memset` may not be the choice in this situation. 😅